molecule/version/converge.yml

```diff
@@ -15,7 +15,8 @@
     - name: Set repo if Red Hat
       ansible.builtin.set_fact:
         ngx_version: -1.23.2-1.{{ (ansible_facts['distribution'] == "Amazon") | ternary('amzn2', ('el' + ansible_facts['distribution_major_version'] | string)) }}.ngx
-        njs_version: -1.23.2+0.7.9-1.{{ (ansible_facts['distribution'] == "Amazon") | ternary('amzn2', ('el' + ansible_facts['distribution_major_version'] | string)) }}.ngx
+        njs_version: -1.23.2+0.7.9-1.{{ (ansible_facts['distribution'] == "Amazon") | ternary('amzn2', ('el' + ansible_facts['distribution_major_version'] | string))
+          }}.ngx
```

Comment:
This needs to go back to how it was before. A single line is much preferable to what it looks like now.